src/main/java/uk/ac/cam/cl/dtg/segue/database/GitDb.java

```diff
@@ -374,8 +345,14 @@ protected JSch getJSch(final OpenSshConfig.Host hc, final org.eclipse.jgit.util.
                     log.info("Fetched latest from git. Latest version is: " + this.getHeadSha());
                 }
             }
-
-
+        } catch (TransportException e) {
+            log.error("Failed to authenticate with the remote content repository via SSH. Ensure the 'Git' section of "
+                    + "segue-config[.cs].properties has valid values, particularly that the key at "
```

Comment:
There's actually a `getPropertiesFile()` method of the Properties object that will give you the file path for the actual in-use file which may be more useful than this generic name?

Reply:
But if not, I'd prefer that the `[.cs]` bit wasn't there, since the API isn't supposed to know about either physics or CS where possible. Just `segue.properties` is fine.